tests/integration-tests/main_test.go

```diff
@@ -0,0 +1,72 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+package integration_tests
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+)
+
+
+var runNumber uint64
+// The appSuffix is a unique string consisting of only characters that are valid app names.
+// This is used to make tests repeatable.
+var appSuffix string
+
+func base26(i uint64) string {
+	result := ""
+	if i == 0 {
+		return "a"
+	}
+	for i > 0 {
+		rem := i % 26
+		i = i / 26
+		result = string(rune('a'+rem)) + result
+	}
+	return result
+}
+
+func calculateRunNumber() error {
+	content, err := os.ReadFile(".run-number")
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			content = []byte("0")
+		} else {
+			return err
+		}
+	}
+	nr, err := strconv.ParseUint(string(content), 10, 64)
+	if err != nil {
+		return err
+	}
+	nextRun := nr + 1
+	os.WriteFile(".run-number", []byte(fmt.Sprintf("%d", nextRun)), 0644)
+	runNumber = nr
+	appSuffix = base26(runNumber)
+	return nil
+}
+
+func TestMain(m *testing.M) {
+	err := calculateRunNumber()
```

Comment:
I'm fine with this change overall.
Please add a bit more explanation why the run-number is making it easier to run locally.
You only seem to be writing it to a file, but not actually using it (or I don't see where you use it).

Reply:
I renamed the method. The run number is indeed not really used but just the app suffix.